### PR TITLE
feat: add support theme for Table component

### PR DESCRIPTION
**Pull request by yvmunayev**
fix: #1226
#1172 

## Changes proposed in this PR:
- feat: add support theme for Table component

- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/nexxtway/react-rainbow/blob/master/CONTRIBUTING.md#submitting-a-pull-request).

@nexxtway/react-rainbow

### CODE REVIEW

File: src/components/Table/head/styled/guideline.js [hunk 1]
```diff
@@ -1,19 +1,28 @@
...
     width: 1px;
 
     :hover {
-        background-color: ${COLOR_BRAND};
+        background-color: ${props => props.brandMainColor};
     }
 
     :active {
-        background-color: ${COLOR_BRAND_ACTIVE};
+        background-color: ${props => props.successMainColor};
```
Comment (LeandroTorresSicilia) [inline src/components/Table/head/styled/guideline.js]: @TahimiLeonBravo  check this styles

File: src/components/Table/head/styled/resizeBar.js [hunk 2]
```diff
@@ -13,18 +22,18 @@ const StyledResizeBar = styled.span`
...
         }
     }
 
     :active {
-        background-color: ${COLOR_BRAND_ACTIVE} !important;
+        background-color: ${props => props.successMainColor} !important;
 
         & span {
-            background-color: ${COLOR_BRAND_ACTIVE};
+            background-color: ${props => props.successMainColor};
```
Comment (LeandroTorresSicilia) [inline src/components/Table/head/styled/resizeBar.js]: @TahimiLeonBravo  same ^^ check styles

**Comment by TahimiLeonBravo** [comment]
@yvmunayev Please review these cases
Screen Shot 2019-12-19 at 12 06 53 PM
Screen Shot 2019-12-19 at 12 07 27 PM
Screen Shot 2019-12-19 at 12 07 04 PM

File: src/components/Table/head/styled/resizeBar.js (hunk 2 above)
Comment (TahimiLeonBravo) [reply src/components/Table/head/styled/resizeBar.js]: @yvmunayev use the `brandDarkColor`

File: src/components/Table/head/styled/resizeBar.js [hunk 3]
```diff
@@ -13,18 +22,18 @@ const StyledResizeBar = styled.span`
...
 
         & span {
-            background-color: ${COLOR_BRAND};
+            background-color: ${props => props.brandMainColor};
         }
     }
 
     :active {
-        background-color: ${COLOR_BRAND_ACTIVE} !important;
+        background-color: ${props => props.successMainColor} !important;
```
Comment (TahimiLeonBravo) [inline src/components/Table/head/styled/resizeBar.js]: @yvmunayev use the `brandDarkColor`

File: src/components/Table/head/styled/guideline.js (hunk 1 above)
Comment (TahimiLeonBravo) [reply src/components/Table/head/styled/guideline.js]: @yvmunayev use the `brandDarkColor`

**Pull request merged**